perma_web/urls.py

```diff
@@ -1,8 +1,11 @@
 from django.conf.urls import url, include
 from django.contrib import admin
+from django.conf.urls import *
```

Comment:
Thanks so much for working on this!!!!

Curious, why do we need both `from django.conf.urls import url, include` and `from django.conf.urls import *`?